src/types/profile.ts

```diff
@@ -23,6 +24,7 @@ export const toInitializedProfile = (
   family_name: user.family_name,
   fiscal_code: user.fiscal_code,
   has_profile: true,
+  is_email_enabled: fromNullable(profile.is_email_enabled).getOrElse(true),
```

Comment:
is there any reason on why `is_email_enabled` is not required in the specs ?

@cloudify @fpersico 

https://raw.githubusercontent.com/teamdigitale/io-functions-commons/v0.5.0/openapi/definitions.yaml#/ExtendedProfile